Vungle/VungleRouter.m

```diff
@@ -765,7 +764,9 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
 {
     NSString *message = nil;
     NSError *playabilityError = nil;
-    if (!isAdPlayable) {
+    if (isAdPlayable) {
+        MPLogInfo(@"Vungle: Ad playability update returned ad is playable for Placement ID: %@", placementID);
```

Comment:
Moving this log to the outside of the delegate check block since QA didn't get a log and thought the callback from the Vungle SDK was never fired (IOS-4052 - not a bug but moving outside for them)